sympy/combinatorics/free_group.py

```diff
@@ -287,6 +293,14 @@ class FreeGroupElm(CantSympify, tuple):
     is_identity = None
     is_AssocWord = True
 
+    _hash = None
+
+    def __hash__(self):
+        _hash = self._hash
+        if _hash is None:
```

Comment:
Neither do I. Actually, I did not notice that this was for `FreeGroupElm` and not for `FreeGroup`.